src/ecmult_impl.h

```diff
@@ -302,6 +377,17 @@ static void secp256k1_ecmult_context_init(secp256k1_ecmult_context *ctx) {
 }
 
 static void secp256k1_ecmult_context_build(secp256k1_ecmult_context *ctx, const secp256k1_callback *cb) {
+#if ECMULT_WINDOW_SIZE <= ECMULT_STATIC_TABLE_MAX
```

Comment:
It'd be nice to have a convenience `#define STATIC_WINDOW_G` that can be tested instead of repeating this expression every time.

Reply:
Will do.